core/adapters/http.go

```diff
@@ -63,6 +63,7 @@ type HTTPPost struct {
 	POST         models.WebURL   `json:"post"`
 	Headers      http.Header     `json:"headers"`
 	QueryParams  QueryParameters `json:"queryParams"`
+	Body         *string         `json:"body,omitempty"`
```

Comment:
Why `*string`, could we just use an empty string `""` to denote absence?

Reply:
It allows us to distinguish between the case of a Post Job with an empty payload, and one which takes its payload from the previous Job.  Thought this might be helpful.